### DISCUSSION TITLE
Wrong itinary calculation on bicycle

### DISCUSSION THREAD

**Discussion by 1Cedric1** [osmandapp/OsmAnd#18571] [General]
Hello dear Osmand team.
I've found a strange behaviour by calculating a bicycle route on the current Android version.
Just see the attached picture.
- The route goes through the mountains instead of the road, using a pedestrian walk.
- The duration of the trip is impossible, so much too short, even on car, as a pedestrian walk is used.
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/61581150/2ad3f0e0-42f1-4b02-bcc1-0183c2f8c03b"  width="300" height="auto"/> </td>
  </tr>
</table>
Hope this will help to make Osmand great!
Kind regards.

**Comment by DmitryAlexei** [comment]
@1Cedric1 hello and thank you for this question.
I can see on your screenshot, that *Use elevation data* option is disabled (for more information see https://osmand.net/docs/user/navigation/routing/bicycle-based-routing#route-parameters---bicycle). This feature can be very useful in mountainous regions. 
You can activate this option through navigation settings or in the main navigation menu (it is located near *Style de conduite* option). 

In the upcoming release, we have further improved the functionality of this feature (*Use elevation data*), route calculation and estimated time of arrival will be more accurate.

**Comment by 1Cedric1** [comment]
Dear @DmitryAlexei , hello and thank you very much for your help. 
You are right, it works now fine with this active option. 
I do not know why I did not see it earlier.
Many thank for your job and take care.
